man/coap-server.txt.in

```diff
@@ -30,9 +31,29 @@ OPTIONS
    Join specified multicast 'group' on startup.
 
 *-p* port::
-   The 'port' on the given address the server will be waitung for connections.
+   The 'port' on the given address will be listening for incoming connections.
+   If (D)TLS is supported, then, then 'port' + 1 will alos be listened on for
+   (D)TLS connections.
    The default port is 5683 if not given any other value.
 
+*-c* certfile::
+   Use the specified PEM file which contains the CERTIFICATE and PRIVATE
+   KEY information. This argument requires (D)TLS with PKI to be available.
+
+*-k* key::
+   Pre-shared key to use for inbound connections. The default key is 'secretPSK' .
```

Comment:
See above for my comment on removing the hard-wired default key for more security.

Reply:
Updated